### PR TITLE
Feature/11425 further platform improvements

### PR DESCRIPTION
**Pull request by gavinwakeley1**
Task: [AB#11425](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/11425)
Task: [AB#11421](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/11421)
Task: [AB#11428](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/11428)
Activity: [AB#11282](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/11282)

### CODE REVIEW

File: src/scripts/bash-nix/tear-down-local.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+context=`kubectl config current-context`
+if [[ "$context" != "docker-desktop" ]]; then 
+  >&2 echo "$context is not a local context!"
+  exit 1
+fi
+
+helm delete bc -n buyingcatalogue
```
Comment (PhysicsJon) [inline src/scripts/bash-nix/tear-down-local.sh:9]: EoF newline

File: src/scripts/pscore/tear-down-local.ps1 [hunk 2]
```diff
@@ -0,0 +1 @@
+helm delete bc -n buyingcatalogue
```
Comment (PhysicsJon) [inline src/scripts/pscore/tear-down-local.ps1:1]: EoF newline

**Review by PhysicsJon — CHANGES_REQUESTED** [review]
Just a couple of EoF newlines

File: src/scripts/pscore/tear-down-local.ps1 (hunk 2 above)
Comment (gavinwakeley1) [reply src/scripts/pscore/tear-down-local.ps1]: Added

File: src/scripts/bash-nix/tear-down-local.sh (hunk 1 above)
Comment (gavinwakeley1) [reply src/scripts/bash-nix/tear-down-local.sh]: Added

**Review by PhysicsJon — APPROVED** [review]

**Pull request merged**